package.json

```diff
@@ -16,11 +16,13 @@
     "export": "npm run build && next export",
     "lint": "tslint --project .",
     "lint-fix": "tslint --fix --project .",
-    "lint-fix-staged": "git diff --staged --name-only | grep -E '\\.tsx?$' | xargs npm run lint-fix && git diff --staged --name-only | grep -E '\\.tsx?$' | xargs git add"
+    "lint-fix-staged": "git diff --staged --name-only | grep -E '\\.tsx?$' | xargs npm run lint-fix && git diff --staged --name-only | grep -E '\\.tsx?$' | xargs git add",
+    "format": "prettier --write \"./**/*.{ts,tsx}\"",
+    "check-format": "prettier --check \"./**/*.{ts,tsx}\""
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint-fix-staged && npm run build-for-husky"
+      "pre-commit": "npm run lint-fix-staged && npm run format && git add"
```

Comment:
I'd like not to use `git add` here. If you have selectively staged things for this commit, that messes it up.

Reply:
Removed!